web/src/user_group_edit.ts

```diff
@@ -309,13 +309,8 @@ function initialize_tooltip_for_membership_button(group_id: number): void {
         ".join_leave_button_wrapper",
```

Comment:
Changes in settings_data.ts for join and leave functions make sure that this function isn't called in the wrong conditions.